### DISCUSSION TITLE
About Node Adapter

### DISCUSSION THREAD

**Discussion by directormac** [arthelokyo/astrowind#188] [Q&A]
Hello i added node adapter and set my output to 'server'
![image](https://github.com/onwidget/astrowind/assets/5866196/b465ae3b-2e77-4459-b9a2-7c6c85209109)
This is what happens now when i try to navigate to /blog other landing pages seems to work fine but not the /blog

**Comment by prototypa** [comment]
Hello @directormac 

The thing is that **AstroWind** is oriented to **static generation**. And you will see that it fails on the blog because the whole blog is built using `getStaticPaths()`. It is what allows access later in the props to certain variables. 

To bring **AstroWind** to server side you will have to make some adjustments. 

Thanks

PD: We are going to include in the Roadmap the task of refactoring AstroWind so that the migration to server is easier. In addition to publishing a guide for it.

**Reply by directormac** [reply]
Thank you! for an update! a guide to customizing and extending every aspect of astrowind would be a lovely addition to this beautiful theme!
